lib/include/tree_sitter/api.h

```diff
@@ -271,6 +280,13 @@ TSTree *ts_parser_parse(
   TSInput input
 );
 
+TSTree *ts_parser_parse_custom(
```

Comment:
If docs are added to the location of my above comment, they should probably be added here too.